protostar/testing/cairo_test_runner.py

```diff
@@ -199,9 +201,11 @@ async def _invoke_test_case(
             program=program,
             state=state,
         )
-        return await StandardTestCaseRunner(
+        runner = StandardTestCaseRunner(
```

Comment:
changes in this file should be reverted 🥶

Reply:
[revert redundant changes](https://github.com/software-mansion/protostar/pull/1363/commits/20ff0c6ba52a828ef2d747e147158aa04c94db4c)